apps/web/pages/api/recorded-daily-video.ts

```diff
@@ -348,11 +166,57 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
       const evt = await getCalendarEvent(booking);
       await sendDailyVideoTranscriptEmails(evt, transcripts);
 
+      return res.status(200).json({ message: "Success" });
+    } else if (req.body?.type === "batch-processor.job-finished") {
```

Comment:
We shouldn't let the code reach till here if body is falsy. We could check this and early return in the handler

Reply:
I am already validating the schema just below if else condition.

Reply:
That's a different thing.

 I mean that we shouldn't be  using optional chaining throughout the code for `req.body.type` or any other prop on body.
 
 We should simply do 
 ```
const body = req.body;
if (!body) {
  // Return early here
}
// Following code will now use just body.
```